guides/source/form_helpers.md

```diff
@@ -158,90 +174,90 @@ Other form controls worth mentioning are text areas, hidden fields, password fie
 <%= form.color_field :favorite_color %>
 ```
```

Comment:
This is a pretty long list. I'm not sure anyone would read these line-by-line.
Maybe we can split it into separate groups?
* common fields like text areas, hidden fields, password fields, number fields
* date and time fields
* miscellaneous fields (sorry, can't seem to think of a better name yet)/